### DISCUSSION TITLE
Add option to configure nodeps from command line (--nodeps) and .ansible-lint

### DISCUSSION THREAD

**Discussion by dmitrij-zavrazhin-1304** [ansible/ansible-lint#4258] [Ideas]
### Use case

We have a use case in which we would like to be able to disable installing content dependencies and performing checks that would fail when modules are not installed in the .ansible-lint config file itself.
Currently the only way to enable this behaviour is by setting the ANSIBLE_LINT_NODEPS environment variable, but that does not fit our use case.

### Proposed Solution

To solve this, we could use an additional 'nodeps' parameter to specify that ansible-lint behaviour should mirror having ANSIBLE_LINT_NODEPS being set to 1.

**Comment by ssbarnea** [comment]
You failed to mention anything about offline mode. What in the offline mode does not suit your needed?

The primary effect of offline mode is that it will not install dependencies.
